Sources/Support/Logging/Convention.swift

```diff
@@ -34,7 +34,7 @@ public struct LogStorageConvention: Sendable {
     /// How is the log file for each executable named
     var executableTargetLogFileNamingStrategy: LogFileNamingStrategy
     
-    public init(baseStorageLocation: BaseStorageLocation, basePathComponents: [String], executableTargetGroupingStrategy: ExecutableTargetGroupingStrategy, executableTargetLogFileNamingStrategy: LogFileNamingStrategy) {
+    init(baseStorageLocation: BaseStorageLocation, basePathComponents: [String], executableTargetGroupingStrategy: ExecutableTargetGroupingStrategy, executableTargetLogFileNamingStrategy: LogFileNamingStrategy) {
```

Comment:
`Convention.init` is intentionally public. The idea of convention is so that the caller *can* create custom conventions as they wish.